docs/self-managed/zeebe-deployment/operations/update-zeebe.md

```diff
@@ -194,6 +198,10 @@ This should resolve automatically once the broker is updated.
 
 If this persists, you may want to [force the update](#rolling-update-is-not-completing).
 
+:::note
+In this specific case, a rollback is possible because users may restart Zeebe with the "skipped" minor version. [Backup and restore](/self-managed/operational-guides/backup-restore/backup-and-restore.md) is possible, but results in partial data loss of everything following the backup. Therefore, a rollback is preferable in this use case.
```

Comment:
:thought_balloon: I think including backup and restore here adds to the confusion, especially because _taking_ a backup is not possible in this state. Restoring from a backup is _always_ possible, no need to mention it here.

Reply:
Updated in latest commit 👍